docs/features/opencdc-record.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'OpenCDC record'
+sidebar_position: 2
+---
+
+An OpenCDC record in Conduit aims to standardize the format of data records exchanged between different connectors within a data processing pipeline. The primary objective is to ensure compatibility between various combinations of source and destination connectors.
+
+## Benefits
+
+1. **Support for Operations**: The format should support representing records for create, update, delete, and snapshot operations.
+1. **Standard Metadata Fields**: Definition of standard metadata fields to provide essential information about each record.
+1. **Integration with Data tools**: We believe that by standardizing on this format regardless of the connector type, we'll easily be able to integrate with any other processing data tool that supported this format.
+
+## Representation
+
+Conduit relies on [protobuf](https://protobuf.dev/) when it comes to defining an OpenCDC record to benefit from the several advantages that it provides. Its definition can be found in the [Buf Schema Registry](https://buf.build/conduitio/conduit-commons/docs/main:opencdc.v1).
+
+When processing records in Conduit, you can always expect this format,
+
+```json
+{
+    "position": "c3RhbmRpbmc=",
+    "operation": "update",
+     "metadata": {
+        "file.path": "./example.in",
+        "opencdc.readAt": "1663858188836816000",
+        "opencdc.version": "v1"
+    },
+    "key": "cGFkbG9jay1rZXk=",
+    "payload": {
+        "before": "eWVsbG93",
+        "after": {
+            "bool": true,
+            "float32": 1.2,
+            "float64": 1.2,
+            "int": 1,
+            "int32": 1,
+            "int64": 1,
+            "string": "orange"
+        }
+    }
+}
+```
+
+:::note
+
+Depending on the operation performed on the record, and the connector used its payload can vary. For example:
+
+- When operation is `create`, there won't be a `payload.before`, but a `payload.after` will.
```

Comment:
```suggestion
- When operation is `create` or `snapshot`, there won't be a `payload.before`, but a `payload.after` will.
```

Reply:
Thank you for the suggestion @maha-hajja !